### ISSUE TITLE
How to calculate the pointer_logits?

### ISSUE THREAD

**Issue by longbowzhang** [manycore-research/PlankAssembly#10]
Dear authors, 
First of all, thanks a lot for this very great work which does inspire us.
I have a question w.r.t the [calculation of the pointer_logits](https://github.com/manycore-research/PlankAssembly/blob/4fa99105b1a966adae763bfaeee6044c41be15d8/plankassembly/models.py#L147-L154).

`pointer_logits = torch.bmm(pointer_feature, outputs.transpose(1, 2))`
`pointer_logits /= self.num_model`

Q: why not divide by `self.num_model` rather than the `sqrt(self.num_model)` which is commonly use such as in the work [polygen?](https://github.com/google-deepmind/deepmind-research/blob/f5de0ede8430809180254ee957abf36ed62579ef/polygen/modules.py#L1362)

Looking forward to your reply!

**Comment by bertjiazheng** [comment]
This is an unintentional implementation. I will look into whether this affects the performance.

**Comment by bertjiazheng** [comment]
Sorry for the delayed response. I had the opportunity to train and test the model. For simplicity, I train and test the model using the complete line as input. The results are as follows.

| Models                     | Precision | Recall | F1    |
|----------------------------|-----------|--------|-------|
| Original implementation         | 92.21 | 91.51 | 91.75 |
| Normalized by `sqrt(num_model)` | 92.66 | 91.73 | 92.10 |

The standard attention normalization shown above does have a minor improvement. Thanks for pointing out this. @longbowzhang

**Comment by bertjiazheng** [comment]
I will close this issue. Feel free to re-open it if you still have any questions. @longbowzhang
